experiments/verify_fasm_to_bels.yaml

```diff
@@ -1,6 +1,15 @@
 designs:
+    - base/
     - basic/
-    # - base/
+    - byu/
+    #- ipassurance_designs/
+    #- lfsr_randgen/
+    #- memories/
+    #- misc/
+    #- ooc/
+    #- riscv/
+    #- simple/
+    #- uart/
```

Comment:
Can you remove these commented out lines?

Reply:
Yes I'll do that.